.github/workflows/frontend-ci.yml

```diff
@@ -31,228 +31,112 @@ jobs:
+        image: bitnami/minio:latest
+        env:
+          MINIO_ROOT_USER: minioAccessKey
+          MINIO_ROOT_PASSWORD: minioSecretKey
+        ports:
+          - 9000:9000
+    env:
+      # API server env vars
+      DJANGO_DATABASE_URL: postgres://postgres:postgres@localhost:5432/django
+      DJANGO_MINIO_STORAGE_ENDPOINT: localhost:9000
+      DJANGO_MINIO_STORAGE_ACCESS_KEY: minioAccessKey
+      DJANGO_MINIO_STORAGE_SECRET_KEY: minioSecretKey
+      DJANGO_STORAGE_BUCKET_NAME: dandi-bucket
+      DJANGO_DANDI_DANDISETS_BUCKET_NAME: dandi-bucket
+      DJANGO_DANDI_DANDISETS_LOG_BUCKET_NAME: dandiapi-dandisets-logs
+      DJANGO_DANDI_DANDISETS_EMBARGO_LOG_BUCKET_NAME: dandiapi-embargo-dandisets-logs
+      DJANGO_DANDI_WEB_APP_URL: http://localhost:8085
+      DJANGO_DANDI_API_URL: http://localhost:8000
+      DJANGO_DANDI_JUPYTERHUB_URL: https://hub.dandiarchive.org/
+      DJANGO_DANDI_DEV_EMAIL: test@example.com
+      DJANGO_DANDI_ADMIN_EMAIL: admin@kitware.com
+      DANDI_ALLOW_LOCALHOST_URLS: 1
+
+      # Web client env vars
+      VITE_APP_DANDI_API_ROOT: http://localhost:8000/api/
+      VITE_APP_OAUTH_API_ROOT: http://localhost:8000/oauth/
+      VITE_APP_OAUTH_CLIENT_ID: Dk0zosgt1GAAKfN8LT4STJmLJXwMDPbYWYzfNtAl
+    steps:
+    - uses: actions/checkout@v4
+
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 20
+
+    - name: Set up Python
+      uses: actions/setup-python@v5
+      with:
+        python-version: '3.11'
+
+    - name: Install web app
+      run: yarn install --frozen-lockfile
+      working-directory: web
+
+    - name: Install latest version of pip
+      run: pip install --upgrade pip
+
+    - uses: actions/cache@v4
+      id: pip-cache
+      with:
+        path: ${{ env.pythonLocation}}/lib/python3.11/site-packages/*
+        key: ${{ env.pythonLocation }}-${{ hashFiles('requirements.txt') }}-${{ hashFiles('setup.py') }}
+
+    - name: Install dandi-api dependencies
+      run: pip install --upgrade --upgrade-strategy eager -e .[dev]
+
+    - name: Apply migrations to API server
+      run: python manage.py migrate
+
+    - name: Create any cache tables
+      run: python manage.py createcachetable
+
+    - name: Install test data
+      run: python manage.py loaddata playwright
+
+    - name: Install dependencies
+      run: yarn install --frozen-lockfile
+      working-directory: e2e/playwright
+
+    - name: Install Playwright Browsers
+      run: npx playwright install --with-deps
+      working-directory: e2e/playwright
+
+    - name: Run Playwright tests
+      run: |
+          # start vue dev server
+          yarn --cwd ../../web/ run dev 2> /dev/null &
+          while ! nc -z localhost 8085; do
+            sleep 3
+          done
+
+          # start the dandi-api server
+          python ../../manage.py runserver &
+
+          # run the tests
+          npx playwright test
+      working-directory: e2e/playwright
```

Comment:
I think we can move the contents of the `e2e/playwright/` directory to the top level `e2e/` directory now.